src/components/AboutUsIcons/AboutUsIcons.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Image from 'next/image';
+
+type AboutUsIconProps = {
```

Comment:
On this project, we mostly use `interface` instead of `type` for declaring a type